### DISCUSSION TITLE
Snapshot for develop branch

### DISCUSSION THREAD

**Discussion by rupinderjeet** [GetStream/stream-chat-android#1650] [Q&A] [answered]
I am trying to use latest 'ui-components' lib in my project. However, most of the customizations I need are not released yet. How can I tell gradle to target this repository's `develop` branch? I searched around, and found that this is what a snapshot is for. Also, I saw that you used something related to  'SNAPSHOTS' in your `build.gradle` when creating/releasing(?) pom.

I have tried creating `aar`(s) for required modules and importing them through "lib" folder. But, I am facing a lot of gradle dependancy-merge conflicts.

So, right now, is there a way to access `develop` branch's content as a snapshot that I can use in gradle? Please let me know.

**Comment by JcMinarro** [comment, marked as answer]
Hello @rupinderjeet 

Currently, we don't have an official Snapshot Deploy Process yet, but for your needs, you can use Jitpack, which allows you to get a compiled version from any OpenSource Github Repository.
Here you can find instructions to use `develop` branch as your version:
https://jitpack.io/#getstream/stream-chat-android/develop-SNAPSHOT

Pay attention that the GroupID needs to be changed to `com.github.getstream.stream-chat-android` and you need to select the proper artifact you want to use.

You can generate a custom version from a concrete SHA1 Commit as well

**Reply by rupinderjeet** [reply]
Thank you. This will help me.

p.s. unintentionally unmarked and marked the answer by mistake. 😰
